sdk/personalizer/test-resources.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+		"baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+		},
+		"location": {
+            "type": "string",
+            "defaultValue": "eastus2euap",
```

Comment:
Is there a specific reason to default to the canary/early access region here?

Reply:
We generally prefer tests to run against the Canary region for a few reasons:
1. Canary is where we deploy the service to first. So it helps to find issues up front.
2. In the past, the test uncovered a bug that caused the service to hang. Having this be in Canary where there are no real customers reduces impact should something like that happen again.